Portal/src/Datahub.Portal/Layout/Topbar/HelpDialog.razor

```diff
@@ -67,11 +73,11 @@
                 }
             </MudItem>
             <MudItem xs="6" Align="Align.End" Class="py-6">
-                @if (_currentStep < 2 && _currentStep > 0)
+                @if (_currentStep < 3 && _currentStep > 0)
                 {
                 <MudButton Type="Button" Variant="Variant.Filled" Color="Color.Primary" EndIcon="@Icons.Material.Filled.ArrowForward" OnClick="NextStep">@Localizer["Next"]</MudButton>
                 }
-                else if (_currentStep >= 2)
+                else if (_currentStep >= 3)
                 {
                     <MudButton Type="Button" Variant="Variant.Filled" Color="Color.Primary" StartIcon="@Icons.Material.Filled.CheckCircleOutline" OnClick="SubmitForm">@Localizer["Submit"]</MudButton>
                 }
```

Comment:
Might be simpler to swap the order of the condition, something like an equal to last step or else?